default-build-spec/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+version: 1
```

Comment:
I suggest `version: 0`, indicating that there is no promise here... just an idea that we will have a version in the future (some very far away future perhaps).

We still have `version: 0` in references, and we likely never will have to formally promise anything about that... which moving to version `1` sort of implies.

Reply:
OK

Reply:
I suspect that "but we said version 0!" is going to be faint comfort when we cause a production outage, but we can try :)